lib/filter/text.js

```diff
@@ -19,7 +19,8 @@ module.exports = Class.create({
 	
 	text: function(opts) {
 		// render text onto canvas
-		// { text, font, size, color, gravity, overflow, marginX, marginY, offsetX, offsetY, characterSpacing, lineSpacing, shadowColor, shadowOffsetX, shadowOffsetY, shadowBlur, outlineColor, outlineThickness, outlineStyle, opacity, autoCrop, background, maxWidth, maxHeight, shrinkWrapPrecision, fontStyle, fontWeight, mode }
+		// { text, font, size, color, gravity, overflow, marginX, marginY, offsetX, offsetY, characterSpacing, lineSpacing, shadowColor, shadowOffsetX, shadowOffsetY, shadowBlur, outlineColor, outlineThickness, outlineStyle, opacity, 
+		, background, maxWidth, maxHeight, shrinkWrapPrecision, fontStyle, fontWeight, mode }
```

Comment:
This line break ain't gonna fly.  You broke a comment into two lines, but the second line has no `//`.